components/motor/gpio/motor_encoder_test.go

```diff
@@ -77,8 +77,8 @@ func MakeIncrementalBoard(t *testing.T) *fakeboard.Board {
 
 func TestCreateMotorWithControls(t *testing.T) {
 	logger := logging.NewTestLogger(t)
-	controlParams := motorPIDConfig{P: 0.2, I: 0.2, D: 0.0}
-	cfg := Config{TicksPerRotation: 100, MaxRPM: 100, ControlParameters: &controlParams}
+	controlParams := &motorPIDConfig{P: 0.2, I: 0.2, D: 0.0}
+	cfg := Config{TicksPerRotation: 100, MaxRPM: 100, ControlParameters: controlParams}
```

Comment:
I'm perfectly fine with this change, but would also have been fine with the old version. Why do this? (Is there something about best practices I should learn from you making this change?)

Reply:
I just added this because `ControlParameters` is now being handled as a pointer. I guess the other way to do it is pass `&controlParams` but without the & in one of the two, it errors